spacepy/lib.py

```diff
@@ -119,11 +124,15 @@ def load_lib():
 def load_call_dict(call_dict, lib):
     """Loads argument/return types from the call dictionary
 
-    @param call_dict: call dictionary. Keyed by function name;
-                      values are [return type, argtype0, argtype 1...]
-    @type call_dict: dict
-    @param lib: library where functions specified in L{call_dict} live.
-    @type lib: ctypes.WinDLL or ctypes.CDLL
+    Parameters
+    ==========
+    call_dict : dict
+        call dictionary. Keyed by function name;
+        values are [return type, argtype0, argtype 1...]
+    
+    lib : ctypes.WinDLL or ctypes.CDLL
```

Comment:
While we're fixing this, like I mentioned before, it's just ctypes.CDLL, no WinDLL anymore.

Reply:
Poke on this one again :)